### ISSUE TITLE
Using safe 1.0.1 & 1.0 Results in `!! permission denied` doing any command outside authentication

### ISSUE THREAD

**Issue by bigorangemachine** [starkandwayne/safe#180]
Hello,

We are using safe with our concourse pipeline.  When I tried to do anything outside authentication I get `!! permission denied`.  I am on OSX 10.14 (Mojave).

How to Replicate:
```
$ safe target https://vault.our-company.org safe -k
$ safe auth github
Authenticating against  at https://vault.our-company.org 
Github Personal Access Token: ******
$ safe paths secret/concourse
!! permission denied
```

Downgrading to 0.9.9 Resolves the issue

**Comment by thomasmmitchell** [comment]
0fabc0968f5c39cdcbd2a0287520c1a084992e27 hopefully addresses this. 1.0.0+ likes to make a call to get what version of kv mount you're trying to hit. I was not really calling the right endpoint to do that - basically, if you didn't have policy rights on sys/*, you'd get rejected.

Works on my machine now, but vault policies are snowflakes... if you'd be willing to take it for test spin so that I can be sure it addresses your use cases, I can provide you a dev binary (over CF slack?)

**Comment by bigorangemachine** [comment]
Sure. Send over slack and I'll download it when I am on wifi

**Comment by thomasmmitchell** [comment]
Fixed in v1.0.2.
